### DISCUSSION TITLE
Is I2S streamig working again with latest build?

### DISCUSSION THREAD

**Discussion by rug1024** [arendst/Tasmota#20346] [Q&A]
Last known working version was 12.5.x
Did some testing in between, I2S streaming was broken.
Can I expect to get this running again? Read something about Arduino Core 3.x, but don't really understand what that is about.

**Comment by s-hadinger** [comment]
It depends what you mean with streaming. I2S audio is working again with the soon to come Arduino Core3 based on esp-idf 5.1. This is required for all new chips.

**Reply by rug1024** [reply]
I am using ESP32-S2 with compiler options USE_I2S_AUDIO, USE_I2S_MIC, USE_SHINE, MP3_MIC_STREAM.
This can still be compiled at the moment, but it's not working. Guess I will have to wait for the new releases with Core3. 
Is there a timelime for Tasmota with this?

**Comment by rug1024** [comment]
Compiled current 13.3.0.3 (ArduinoCore2) with Gitpod. I2S-Streaming works nicely again.
With Core3 it does not yet work, but will probably be solved as development evolves.

**Comment by rug1024** [comment]
Some time has passed, Core2 support has now been removed, but still I2S-Streaming does not work when compiled with Core3.
